core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -54,6 +54,9 @@
 	
 	/** The balls the canon will shoot. */
 	private ArrayList<Ball> cannonBallList = new ArrayList<Ball>();
+	
+	/** The roof hitbox offset. */
+	private float ROOF_OFFSET = 10;
```

Comment:
This is only useful for the roof hitbox, it wont be used anywhere else. So best to keep it here.